CK2Plus/common/defines/000_CK2Plus_defines.lua

```diff
@@ -14,4 +14,62 @@ NDefines.NMilitary.SHATTERED_RETREAT_MAX_PROVINCES = 5
 -- Tribal Empty Holding bonuses (this may need tweaked later)
 NDefines.NCharacter.TRIBAL_EMPTY_HOLDING_TAX_MULTIPLIER = 0.5
 NDefines.NCharacter.TRIBAL_EMPTY_HOLDING_LEVY_MULTIPLIER = 0.1
-NDefines.NCharacter.TRIBAL_EMPTY_HOLDING_GARRISON_MULTIPLIER = 0.9
\ No newline at end of file
+NDefines.NCharacter.TRIBAL_EMPTY_HOLDING_GARRISON_MULTIPLIER = 0.9
+-- Changes for combat tactics overhaul

```

Comment:
```suggestion

-- Changes for combat tactics overhaul
```
Wouldn't let me do this during the merge. Fixing it like this.